PositionalContent.Code/Web/UI/App_Plugins/PositionalContent/contentEditor/positionalcontenteditor.controller.js

```diff
@@ -163,7 +163,13 @@ angular.module("umbraco").controller("HiFi.PositionalContentDialogController", [
 
         };
 
```

Comment:
I know you're being safe here but because v8 changes to editorService I think  it can just be

 $scope.close = function () {
     editorService.close();
 }